TwilioVerifyDemo/TwilioVerifyDemo.xcodeproj/project.pbxproj

```diff
@@ -602,7 +602,7 @@
 				MARKETING_VERSION = 1.0;
 				PRODUCT_BUNDLE_IDENTIFIER = com.twilio.TwilioVerifyDemo;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "";
+				PROVISIONING_PROFILE_SPECIFIER = "match AdHoc com.twilio.TwilioVerifyDemo";
```

Comment:
Can we set the provisioning profile when building the apps?

Reply:
It's needed, we need to set a provisioning profile with the push capability and with it the team